apps/webapp/app/components/primitives/ShortcutKey.tsx

```
@@ -34,17 +36,17 @@ export function ShortcutKey({ shortcut, variant, className }: ShortcutKeyProps)
       {modifiers.map((k) => (
-        <Fragment key={k}>{modifierString(k, isMac)}</Fragment>
+        <span key={k}>{modifierString(k, isMac)}</span>
       ))}
-      {character}
+      <span>{character}</span>
     </span>
   );
 }
 
-function keyString(key: String, isMac: boolean, size: "small" | "medium") {
+function keyString(key: String, isMac: boolean, variant: "small" | "medium" | "medium/bright") {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)